app/src/main/res/layout/activity_main.xml

```diff
@@ -1,19 +1,39 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout
+<RelativeLayout android:id="@+id/relative_main"
     xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context="com.khanhhuy.movie_db.MainActivity">
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    
+    <android.support.design.widget.TabLayout
```

Comment:
use tablayout with viewpager is better

Reply:
https://developer.android.com/reference/android/support/design/widget/TabLayout.html